src/parse-mockdb.js

```diff
@@ -538,6 +538,11 @@ function includePaths(object, pathsRemaining) {
 function fetchObjectByPointer(pointer) {
   const collection = getCollection(pointer.className);
   const storedItem = collection[pointer.objectId];
+
+  if (storedItem === undefined) {
```

Comment:
No, this is correct.

What I want to check is if the collection contained an item with that ID. If it does not `storedItem` will be `undefined`, not `null` (and `undefined !== null`).

`if storedItem == null` would work but is not as precise.

Actually, I think the collection should not contain null values anyway, right? So doing the relaxed check would hide potential errors.